jekyll-exporter.php

```diff
@@ -35,6 +35,32 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
  */
 
+// Extra options
+
+$config = array();
+
+// $config["site_url"] = "https://mywebsite.fr/"; // Destination site URL
+// $config['force_dest_dir'] = "_export/"; // Force destination directory to relative one (without temp folder, will deactivate zip)
+
+$config['forceHtml'] = false; // true;	// Force export to HTML
+
+// Skip some parts of the script that you do not need
+// $config['skip_options'] = true; 
+// $config['skip_posts'] = true; 
+// $config['skip_uploads'] = true; 
+// $config['skip_comments'] = true; 
+
+$config['comment_target_data'] = true;		// Export comments as data files
+// $config['comment_target_pages'] = true;	// Export comments as jekyll pages
+// $config['comment_target_github'] = true; // Export comments to GitHub discussions (for use with Giscus)
+
+// To export comments to GitHub for Giscus (https://giscus.app)
+// $config['gh_token'] = "gho_xxxx";					 // Get token from command line: gh auth token 
+// $config['gh_repo'] = "R_kgDOIVZ3BQ";				 // Get values from giscus config page
+// $config['category_id'] = "DIC_kwDOIVZ3Bc4CSRMx"; // Get values from giscus config page
```

Comment:
Same questions as above, if (A) the site URL can be retrieved programmatically, and if options can be configured via the web interface to avoid modifying the source code directly.

Reply:
About the web interface options, I am not sure if this is really a good thing to invest time on that:
- exporting by the web interface seems to work only on small WordPress site, else it will fail ; I guess this is why there are some bad reviews on the WordPress plugin page
- you can assume that people moving to Jekyll won't be afraid to modify a config file (else they probably should not move to Jekyll ;))